src/finite.jl

```diff
@@ -293,7 +293,13 @@ const BalancedAccuracyType = API.FussyMeasure{
     human_name="balanced accuracy",
 )
 
-register(BalancedAccuracy, "balanced_accuracy", "bacc", "bac")
+register(
+    BalancedAccuracy,
+    "balanced_accuracy",
+    "bacc",
+    "bac",
+    "Ppprobability_of_correct_classification",
```

Comment:
This is misspelled I believe. "Ppp" -> "p"

Reply:
Yeah, I also caught my error and fixed it in a hot fix: https://github.com/JuliaAI/StatisticalMeasures.jl/pull/24/files